blocks/library/embed/index.js

```diff
@@ -96,6 +99,10 @@ function getEmbedBlockSettings( { title, icon, category = 'embed', transforms, k
 				this.unmounting = true;
 			}
 
+			setIsSelected() {
+				this.props.setIsSelected( true );
```

Comment:
The only thing bothering me with this PR is this prop, I mean, if we could avoid it by bubbling the focus event or something, it would have been great. But I understand it might be tricky.

Reply:
>I mean, if we could avoid it by bubbling the focus event or something, it would have been great.

That would be interesting. Let me play with it a bit to see if I can get it to work.

Reply:
>>I mean, if we could avoid it by bubbling the focus event or something, it would have been great.
>
>That would be interesting. Let me play with it a bit to see if I can get it to work.

Luckily it _just works_ to dispatch a custom event. See 8a6c493. Simplifies things a fair bit.